### ISSUE TITLE
axInput: ngModelOptions forward compatibility breaks focusout behavior in Angular >= 1.3

### ISSUE THREAD

**Issue by alex3683** [LaxarJS/laxar_uikit#97]
Within `input.js` the function `initializeNgModelOptions()` was introduced to anticipate the `ngModelOptions` directive introduced in AngularJS 1.3. Tests with AngularJS 1.3 showed that this breaks in case of `updateOn: 'focusout'`. Thus a version check should be introduced like this:

```
if( ng.version.major < 1 || ( ng.version.major === 1 && ng.version.minor < 3 ) ) {
   initializeNgModelOptions();
}
```

**Comment by x1B** [comment]
We should probably remove the forward-compatibility behavior for LaxarJS `v1.0.0`

**Comment by alex3683** [comment]
If we set the minimum required AngularJS version to 1.3.x, this would be fine.

**Comment by x1B** [comment]
Marking _enhancement_ since LaxarJS isn't officially compatible with AngularJS 1.3.x (we should change that).

**Comment by x1B** [comment]
Marking _enhancement_ since LaxarJS isn't officially compatible with AngularJS 1.3.x (we should change that).

**Comment by x1B** [comment]
_Marking question:_ This depends on (#98) (`axInput` and its tests are incompatible with AngularJS 1.3) and _may_ require the _breaking change_ of abandoning AngularJS `1.2.x` compatibility and with it the support for MSIE8.

**Comment by x1B** [comment]
A preliminary compatibility switch has been implemented on commit 1718a19

**Comment by x1B** [comment]
_Unmarking question:_ forward compatibility will be removed with LaxarJS 1.0 (only AngularJS 1.3.x will be supported).

**Comment by x1B** [comment]
@alex3683 would you like to take a look? commit 7722f87

**Comment by alex3683** [comment]
Looks good and works as expected.

**Comment by x1B** [comment]
Implemented on `master` (`v1.0.0`).
